src/util-logopenfile.h

```diff
@@ -169,7 +172,7 @@ LogFileCtx *LogFileNewCtx(void);
 int LogFileFreeCtx(LogFileCtx *);
 int LogFileWrite(LogFileCtx *file_ctx, MemBuffer *buffer);
 
-LogFileCtx *LogFileEnsureExists(LogFileCtx *lf_ctx);
+LogFileCtx *LogFileEnsureExists(int thread_id, LogFileCtx *lf_ctx);
```

Comment:
should thread_id be an `uint32_t` here? Btw, perhaps it's good to add a type(def) for `ThreadId` or similar